examples/miscellaneous/multiple_dropfile.py

```diff
@@ -0,0 +1,48 @@
+from kivy.app import App
+from kivy.uix.button import Button
+from kivy.core.window import Window
+from kivy.uix.boxlayout import BoxLayout
+
+
+class DropFile(Button):
+    def __init__(self, **kwargs):
+        super(DropFile, self).__init__(**kwargs)
+
+        # get app instance to add function from widget
+        app = App.get_running_app()
+
+        # add function to the list
+        app.drops.append(self.on_dropfile)
+
+    def on_dropfile(self, widget, filename):
+        # a function catching a dropped file
+        # if it's dropped in the widget's area
+        if (self.pos[0] < Window.mouse_pos[0] < self.pos[0] + self.width and
```

Comment:
but only if he'd at least see something in the console when any file is dropped, otherwise it shouldn't matter (tested with both py2 and py3).